proposals/wac-ucr/index.bs

```diff
@@ -1114,13 +1114,178 @@ and the presentation goes off perfectly.
 Requirements {#requirements}
 ================================================================================
 
-Issue: Populate requirements based on established use cases.
+## Functional Requirements ## {#req-functional}
+
+1. The system shall permit or restrict an [=agent's=] ability to access a 
+    [=resource=] or [=collection=]. [[#uc-basic]], 
+    [[#uc-collections]], [[#uc-blockagents]]
+
+1. The system shall allow access to be limited based on the [=identity=] of 
+    the [=agent=]. [[#uc-basic]], [[#uc-collections]]
+
+1. The system shall allow access to be limited to an [=agent=] 
+    based on the [=agent's=] membership in a certain group of [=agents=]. 
+    [[#basic-group]]
+    
+1. The system shall allow access to be limited to an [=agent=] 
+    based on the client [=application=] in use by the [=agent=]. 
+    [[#uc-trusted-applications]]
+
+1. The system shall allow for access to a certain resource to be limited to 
+    the [=agent=] that created that resource.
+    [[#collection-readappendwrite]]
```

Comment:
"allow for access" is a bit vague.. unclear if something like control access would be part of the solution.

also the UC is focused on the collection resource.

Is "certain" superfluous?

Reply:
> "allow for access" is a bit vague.. unclear if something like control access would be part of the solution.

This isn't specific to a particular access mode, but instead indicates that in place of a specific agent identity, you can provide a type (i.e. AgentClass) that would indicate the creator of the resource.

> also the UC is focused on the collection resource.

The rule would need to be set at the collection level because it is identifying a permission that would be assigned upon the creation of a resource. It could apply to a created collection or resource though (updated)

> Is "certain" superfluous?

Yes. Removed

https://github.com/solid/authorization-panel/pull/152/commits/80d4d56b33a6e9b40b9d0469939d40c7f9e7a04f

Reply:
I meant that the cited UC is about a collection but the description reads as though it is for a non-collection resource. If you centre it around "collection", don't need to mention "resource" as well (re current upadate).

Reply:
Both are applicable. Additional adjustments for clarity in https://github.com/solid/authorization-panel/pull/152/commits/01470665d30ec69832621b785605c42c3fc522b9